book/integration-service.md

```diff
@@ -205,6 +205,10 @@ The Integration service needs secrets mounted so that the `Environment Provision
 16. Repeat step 10
 17. Repeat step 11-13
 
+### Image extraction details
+
+Integration service is getting specific information about image being build by build-definition using [those functions](https://github.com/redhat-appstudio/integration-service/blob/06a2947061c943ea9a28a1a39957737fcf8bdc92/tekton/utils.go#L100-L146). Each one of them corresponds with the [results](https://github.com/redhat-appstudio/build-definitions/blob/5881b7381ec2efe226ddcfb6c19a8a9e71e68166/pipelines/template-build/template-build.yaml#L274-L281) that are being reported by build-definitions.
```

Comment:
Maybe we can list the result names and what they represent? 
We can also have a blurb on how the Integration service uses those results - e.g. the `IMAGE_URL` and `IMAGE_DIGEST` are used to construct the component image reference, and `CHAINS-GIT_URL` and `CHAINS-GIT_COMMIT` are added to the source section of the component within the Snapshot.

Reply:
Agreed, will include results with their usage in IS.